.pre-commit-config.yaml

```diff
@@ -26,3 +26,12 @@ repos:
     rev: 6.1.0
     hooks:
     -   id: flake8
+-   repo: https://github.com/codespell-project/codespell
+    rev: v2.1.0
+    hooks:
+    - id: codespell
+      name: 'Spell checker'
+      entry: codespell
+      language: python
+      exclude: '^frontend/(build|node_modules)/|.*\.lock$'
+      args: ['--ignore-words', '.codespellignore']
```

Comment:
Adjust spacing to others.
It means, 
- 3 spaces between `-` and `id`
- Match the start of the lines `name`, `entry`, ...  with the `id`'s.

Reply:
Done!